test/snippet/core/detail/popcount.cpp

```diff
@@ -8,10 +8,10 @@ int main()
     uint32_t t2 = 0b0000'0000'0000'0000'0000'0000'0000'0000;
     uint64_t t3 = 0b1000'0000'1111'0000'0000'0000'0000'0110'0000'0000'0000'0000'1110'0000'0000'0001;
 
-    seqan3::debug_stream << seqan3::detail::popcount(t0) << '\n'; // 4
-    seqan3::debug_stream << seqan3::detail::popcount(t1) << '\n'; // 7
-    seqan3::debug_stream << seqan3::detail::popcount(t2) << '\n'; // 0
-    seqan3::debug_stream << seqan3::detail::popcount(t3) << '\n'; // 11
+    seqan3::debug_stream << std::popcount(t0) << '\n'; // 4
```

Comment:
All of these snippets, are they used at all, now that all of it has moved to `std`? 
I guess they should either be moved into the std folder or removed entirely?